README.md

```diff
@@ -34,3 +41,19 @@ Ex:
     ...
   }
 }
+```
+
+If the only the guild ID is known, packets can be published as a `SEND` event. The gateway which receives this packet is then reponsible for calculating the correct shard ID and re-publishing it in the above format. This format is available so that workers do not need to be aware of the shard count (required when calculating shard ID from guild ID).
```

Comment:
small typo "If the only the guild ID is known" => "if only the guild ID is known"